docs/diagrams/DoneClassDiagram.puml

```diff
@@ -0,0 +1,71 @@
+@startuml
+class DoneCommandParser implements Parser {
+    /' Methods '/
+    + {static} parse(args : String) : DoneCommand
+}
+
+interface Parser<T> {
+    parse(userInput : String) : DoneCommand
+}
+
+abstract class Command {
+    {abstract} execute(Model model) : CommandResult
+}
+
+class DoneCommand extends Command {
```

Comment:
There might be a dependency between DoneCommandParser and DoneCommand as changing DoneCommand would result in some changes in DoneCommandParser.